vignettes/view-composition.Rmd

```diff
@@ -0,0 +1,186 @@
+---
+title: "View Composition"
+author: "Alicia Schep"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_document
+editor_options: 
+  chunk_output_type: inline
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Combining multiple data displays
+
+Vegalite and Altair have several methods to help combine different views of these data.  
+
+* [**Layering**](#layer): Combine several displays on the same set of axes
+* [**Horizontal Concatenation**](#hconcat): Position several displays next to each other, horizontally.
+* [**Vertical Concatenation**](#vconcat): Position several displays next to each other, vertically
+* [**Faceting**](#facet): Create panels showing the same view of different subsets of the data
+* [**Repeating**](#repeat): Create panels that differ in the encoding of one or more channels
+
+For the first three methods, Altair has functions that take in two chart objects. Those chart objects can either be a simple chart made by `alt$Chart`, or a compound chart made by any of the methods above.  Altair and this R package also provides operators for these methods:
+
+| Operation  | Function Syntax | Operator Syntax |
+| ------------- | ------------- | ------------- |
+| Layering  | `alt$layer(chart1, chart2)`  | `chart1 + chart2`  |
+| Horizontal Concatenation  | `alt$hconcat(chart1, chart2)`  | `chart1 | chart2`  |
+| Vertical Concatenation  | `alt$vconcat(chart1, chart2)`  | `chart1 & chart2`  |
+
+In contrast, faceting and repeating are methods for Chart objects, so if you had created a chart called `chart1` you would call them using `chart1$`:
+
+| Operation  | Function Syntax |
+| ------------- | ------------- |
+| Faceting | `chart1$facet(...)`  |
+| Repeating  | ``chart1$`repeat`(...)``  |
+
+Note that for "repeat", backticks are required around the word `repeat`. This is because `repeat` is a [reserved word](http://stat.ethz.ch/R-manual/R-devel/library/base/html/Reserved.html) in R!
+
+## Layered Charts {#layer}
```

Comment:
I don't know if this should be `## Layered charts {#layer}` to match the previous heading (which - I think - is consistent with the other vignettes).